README.md

```diff
@@ -0,0 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# portfolio project
```

Comment:
- [ ] It would be better to Capitalize each word in the Project title since this is a header. For example: **Portfolio Project**